docs/blog/posts/content-tags.md

```diff
@@ -0,0 +1,24 @@
+---
+authors:
+  - Carlos
+category:
+  - Admin App
+  - Api
+date: 2024-06-17
+---
+
+# New feature detected: Tags
+
+Filtering contents has been made easier with tags.
+
+<!-- more -->
+
+## Manage tags
+
+On the Admin App, it is now possible to add tags to contents to categorise them.
+
+You can create a tag, add it the content, or delete it directly within the edit content page. The "Tags" bar shown on the image below will allow you to quickly manage tags.
+
+![edit page tags UI screenshot](../images/tags-edit.png){: .blog-img }
```

Comment:
After making the delete button right-aligned, can you take another screenshot? 

Also (nitpick) call the tags "TAG 1" and "TAG 2" for consistency. For the title also say "title text" or smt instead of add Tags.